build/build.py

```diff
@@ -48,7 +48,10 @@ def is_windows():
 
 
 def shell(cmd):
-  output = subprocess.check_output(cmd)
+  try:
+    output = subprocess.check_output(cmd)
+  except subprocess.CalledProcessError as e:
+    print(e.output)
```

Comment:
I think we should reraise the exception here?

Reply:
Done

Reply:
I think to reraise you can just write `raise` with no `e`.

Reply:
done